src/SpendPermissionManager.sol

```diff
@@ -427,8 +427,8 @@ contract SpendPermissionManager is EIP712 {
             revert SpendTokenWithdrawAssetMismatch(spendPermission.token, withdrawRequest.asset);
         }
 
-        // check spend value equals withdraw request amount
-        if (value != withdrawRequest.amount) {
+        // check spend value is not less than withdraw request amount
+        if (value < withdrawRequest.amount) {
             revert SpendValueWithdrawAmountMismatch(value, withdrawRequest.amount);
```

Comment:
Maybe it's just me, but I find it easier to reason about withdraw amount > value being a problem than value < withdraw amount?

```suggestion
        // check withdraw request amount does not exceed spend value
        if (withdrawRequest.amount > value) {
            revert WithdrawAmountExceedsSpendValue(withdrawRequest.amount, value);
```